lib/Service/MailTransmission.php

```diff
@@ -142,8 +149,12 @@ public function sendMessage(Account $account, LocalMessage $localMessage): void
 			$mail->send($transport, false, false);
 			$localMessage->setRaw($mail->getRaw(false));
 		} catch (Horde_Mime_Exception $e) {
-			$localMessage->setStatus(LocalMessage::STATUS_SMPT_SEND_FAIL);
 			$this->logger->error($e->getMessage(), ['exception' => $e]);
+			if (in_array($e->getCode(), $this->retriableCodes)) {
+				$localMessage->setStatus(LocalMessage::STATUS_SMPT_SEND_FAIL);
+				return;
+			}
+			$localMessage->setStatus(LocalMessage::STATUS_ERROR);
 			return;
```

Comment:
this seems like the essential *fix* that we can also backport. It makes recoverable errors retryable automatically. Could we extract/reduce this as the backportable part and have the copy to sent mailbox *feature* only for main/3.7?